spec/services/metrics/taxonomy_metrics_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+require 'gds_api/test_helpers/content_store'
+
+include ::GdsApi::TestHelpers::ContentStore
+
+module Metrics
+  RSpec.describe TaxonomyMetrics do
+    describe '#count_taxons_per_level' do
+      before do
+        content_store_has_item('/', root_taxon.to_json, draft: true)
+        content_store_has_item('/taxons/root_taxon', multi_level_child_taxons.to_json, draft: true)
```

Comment:
Might be good to use the newer "level one" terminology, especially given you went to the effort of making that change elsewhere.

Reply:
fixed